store/myevent.go

```diff
@@ -0,0 +1,68 @@
+package store
```

Comment:
I assume that this file is created for test purposes ?  If it is not written for test purposes why it is called `myevent.go`

Reply:
Yeah it is :)